Documentation/Cluster.rst

```diff
@@ -75,30 +79,30 @@ Removing a node from the cluster
 To the remove a node from the cluster, run:
 
   ::
-    
-    sudo mcvirt cluster remove-node --node <Remote Node Name>
-    
+
+    mcvirt cluster remove-node --node <Remote Node Name>
+
 
 Get Cluster information
 -----------------------
 
 * In order to view status information about the cluster, use the 'info' parameter for MCVirt, without specifying a VM name::
 
-    sudo mcvirt info
+    mcvirt info
 
 
 Off-line migration

```

Comment:
Sorry to be a pain - would you be able to rename this to 'Virtual Machine Migration' and stick a note about the '--online' flag that can perform online migration